### ISSUE TITLE
Canape: Menu Images on Homepage don’t link to Related Pages

### ISSUE THREAD

**Issue by AMcGrady** [Automattic/themes#630]
## Steps to replicate

In wp-admin click Food Menus → Menu Sections to create sections like ‘Appetizers’ or ‘Main Courses’ and assign them titles and descriptions.

Click Add One Item or Add Many Items to add individual dishes to your menu. Changing the order of items in the menu is as easy as dragging and dropping them up and down.

Click on My Site → Pages to create a new page and assign the Menu template to it. Your menu sections and the menu items within them will automatically show up on that page.

Visit Customize → Theme Options and choose ‘Front Page.’

For each Featured Menu Section in the panel, choose an existing Menu Section and upload an image. 

Save your changes and check your homepage – it will display your chosen image as well as your section description, if you’ve added one. 

Clicking on the images on the homepage should now should take you to the related Menu Items page.

## Result

Clicking on the images on the homepage leads to a "Oops. The page could not be found"message.

## Expected

Expected the Menu Section Pages to be found and displayed.

I'm not sure if this error is universal or specific to site https://pizzeriacomparsa.com.

* User report: [[#10552421-hc](https://mc.a8c.com/support-stats/happychat/transcript.php?id=10552421)]
* Follow-up ticket: [#[1815475-zd](https://woothemes.zendesk.com/agent/tickets/1815475)]

**Comment by laurelfulford** [comment]
Thanks for the report, @AMcGrady! 

I went looking for a weird discussion I had about this site and Canape a while ago, and found there was another one yesterday, too! 

p1550162274121100-slack-themes
p1550619034228800-slack-themes

This seems site-specific to me, and possibly more to do with the Nova Menu than the theme, but leaving here at least until we have more information.

Was also posted here - more info may come up there, too: p2EDhh-Fv-p2

**Comment by alaczek** [comment]
Yep, I concur that this is probably a weird issue on the intersection of Food Menus CPT and this particular site.

**Comment by AMcGrady** [comment]
With suspicion that this is "a weird issue on the intersection of Food Menus CPT and this particular site." what do you recommend as the next step?

My thoughts are to either have the user delete and recreate their Menu in wp-admin or to move this post to JPOP to investigate the Food Menus CPT side of things.

Thoughts?

**Comment by alaczek** [comment]
Yep, I think trying to re-create the food menus from wp-admin (and maybe make sure there are no posts/pages with the same title just to be safe) is a good suggestion. If the issue still persists after, move to JPOP to investigate.

Were you able to replicate the issue on your own test site?

**Comment by damiannep** [comment]
The issue seems to be that the home page has the menu template. This doesn't actually seem to be a bug, but I could be wrong here. Creating a new page to be the home page (not with the menu template) resolves the issue. I will reply to the user to confirm that the issue is resolved..

**Comment by eoinsav** [comment]
Just noting here for the sake of posterity that I had the same issue in #10997493-hc (follow-up here: #1877425-zen). The pages for the menu sections and items were not loading until I deleted the homepage. I have no idea why.

**Comment by tbradsha** [comment]
Another here: #7719376-hc, #1981678-zen

I created a new page, switched it to the new homepage, and it worked as expected. I then switched it back to the old page, and the links worked again, so I was able to delete the new page and keep the old one.

**Comment by samiff** [comment]
Like above, only way I could get this working was to unset the static homepage in `Customize > Homepage Settings` (which was using the default page template), and then set the homepage back. 

12684350-hc

**Comment by khristiansnyder** [comment]
Same as above. Switching to a different homepage and then back again helps bring back the food items.

3743552-hc

**Comment by sophiegyo** [comment]
I experienced this in 30188421-hc.

I found this GH issue, and tested un-setting, then re-setting the static front page. This resolved the issue and now when I click on featured menu sections on the front page, they load correctly.

**Comment by marcuswisecaesar** [comment]
Experienced this here - 36943138-hc (follow-up 5560046-zen)

Found this GH issue, tested un-setting, then reset the static front page. Nothing happened.  Clicked on Appearance → Menus and was shown the Food Menu item in the WP Admin menu.

**Comment by happychait** [comment]
7393664-zen

Fixed by switching the home page and switching it back to the original home page.

**Comment by ezebecke** [comment]
7865078-zen fixed homepage featured sections with mentioned workaround
